### PR TITLE
Added version service in flyteadmin

### PR DESCRIPTION
**Pull request by evalsocket**
Signed-off-by: Yuvraj <evalsocket@gmail.com>

# TL;DR
Added version proto for flyteadmin

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Tracking Issue
https://github.com/lyft/flyte/issues/173

### CODE REVIEW

File: protos/flyteidl/admin/version.proto [hunk 1]
```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package flyteidl.admin;
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin";
+
+// Represents a request structure to get version of flyteadmin.
+message Version {
```
Comment (kumare3) [inline protos/flyteidl/admin/version.proto:7]: can we follow this structure - https://github.com/flyteorg/flytestdlib/blob/master/version/version.go#L14-L21
Have 
Build, Version, BuildTime

File: protos/flyteidl/admin/version.proto [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    // Specifies the GIT sha of the build
+    string	Build = 1;
+
+    // Version for the build, should follow a semver
+    string Version = 2;
+
+    // Build timestamp
```
Comment (katrogan) [inline protos/flyteidl/admin/version.proto:14]: how about google.protobuf.Timestamp as the type here?

File: protos/flyteidl/admin/version.proto [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+package flyteidl.admin;
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin";
+
+// Represents a request structure to get version of flyteadmin.
+message Version {
+    // Specifies the GIT sha of the build
+    string	Build = 1;
+
+    // Version for the build, should follow a semver
```
Comment (katrogan) [inline protos/flyteidl/admin/version.proto:11]: I wonder if it be worthwhile to add a proper semver message or keep with the parsable string convention?

File: protos/flyteidl/admin/version.proto (hunk 2 above)
Comment (evalsocket) [reply protos/flyteidl/admin/version.proto:14]: I am not sure. I tried `google.protobuf.empty` and my proto gen(pbjs) failed. I can try but not sure https://github.com/flyteorg/flyteidl/pull/106/files#diff-9324a86791de0a6631e4fd9acccbe51c4b5f06c33180fd732c69de560e363246R493

File: protos/flyteidl/admin/version.proto [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    string Version = 2;
+
+    // Build timestamp
+    string BuildTime = 3;
+}
+
+// Represents a empty version request.
+message EmptyVersionRequest {
```
Comment (kumare3) [inline protos/flyteidl/admin/version.proto:19]: no need to call it EmptyVersionRequest.
Lets call it GetVersionRequest - thats the convention

File: protos/flyteidl/admin/version.proto (hunk 3 above)
Comment (kumare3) [reply protos/flyteidl/admin/version.proto:11]: @katrogan this is the auto-injected version into the binary, everytime anyone builds it. this is better than just semver, as someone may be running their own version

**Pull request closed without merging**